pkg/image/registry/imagestream/strategy.go

```diff
@@ -487,6 +488,79 @@ func (s Strategy) Decorate(obj runtime.Object) error {
 	return nil
 }
 
+// Export prepares the object for exporting.
+func (Strategy) Export(obj runtime.Object, exact bool) error {
+	is, ok := obj.(*api.ImageStream)
```

Comment:
This is the most contentious one, add a todo here that we need to export a usable image stream.  I'm going to come back in and try to clean it up